dsa/CustomGraph.java

```diff
@@ -0,0 +1,110 @@
+import java.util.ArrayDeque;
+import java.util.ArrayList;
```

Comment:
You are not supposed to use inbuilt classes in Module-1, implement it from scratch.

Reply:
Okay sir, sure i will correct it.